gridstatus/caiso.py

```diff
@@ -1466,6 +1466,17 @@ def _get_historical(
     verbose: bool = False,
 ) -> pd.DataFrame:
 
+    """Get the historical data file from CAISO given a data series name, formats, and returns a pandas dataframe.
+
+    :param file: The name of the data we are wanting, which is equivalent to the file to get from CAISO
+    :type file: str
```

Comment:
I don't have strong feelings about which to use, but I think we should standardize on function doc strings. [Here are a few options](https://stackoverflow.com/a/24385103/5755357) of which I slightly prefer Google's.

Reply:
Oop good catch, I also prefer google's (with types) and forgot to update my autodocstring extension. 

![CleanShot 2024-09-30 at 15 24 15@2x](https://github.com/user-attachments/assets/f989b021-11f4-41e5-a105-32eedf39ad11)

Highly recommend this though. https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring